### DISCUSSION TITLE
Preprint illustrating the use of pi-Base to investigate T1-not-T2 properties

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#432] [Show and tell]
I'm interested in community feedback on this preprint:

https://www.overleaf.com/read/vrndwjxgqnfb

I'm fairly confident in the mathematical results as they have already mostly been aired out between here and the StackExchange network.

The last section includes a bit of opining of how pi-Base and other tech changes the way we do math, and how we should be reconsidering the reward structure for math researchers. These are just my opinions, but of course I'm interested in the perspectives of other community members.

**Comment by prabau** [comment]
I am impressed at how this turned out.  The paper has a good story line, is engaging, and kind of hints at the pace of discovering all these results.  Good show.

If you want, I can share some minor comments of detail on a few things.  But really no negative feedback on any of this.

**Reply by StevenClontz** [reply]
Thanks for the kind words. Any comments you want to share are appreciated.

**Comment by prabau** [comment]
Def 1: seems it would be better to make more explicit the implied universal quantifier, maybe "every convergent sequence has ...", or in the plural "convergent sequences ..."

Sentence before Def 4: maybe it would be more honest to say [16] introduced the notion and showed it to be equivalent to the following def.  Maybe does not matter, as things get clarified in the "k-Hausdorff, revisited" section.

paragraph after Thm 5: isn't S74 more generally known as the Niemytzki plane?

proof of Theorem 17: "The diagonal of a space is homeomorphic to itself": does this phrasing make sense?

Def 22: leaves to be desired.  Not entirely clear what is closed in what. (compared with Def 24 for example, which is totally clear)

Sentence before Def 40: the property was not introduced in [3], but instead in https://doi.org/10.1090/proc/13318.

Def 50: was the term "anti-locally compact" made up for this article?  Annoyingly, it's unrelated to the terminological scheme from Bankston - "the total negation of a topological property" (https://doi.org/10.1215/ijm/1256048236).

Sentence before Def 54: locally Hausdorff was definitely not "introduced" in [6].  For example, see Niefield - "A note on the locally Hausdorff property", already from 1983, and surely there are other papers before that.

References: lots of entries have miscapitalized words (e.g. US, KC, A, B, etc, etc).  Also, giving the user id from users in mathse/mo does not seem to add value, as it can be obtained just from clicking to the post.  (and what's the point especially for mathematically and otherwise semi-illiterate mathse posts from people like in [8] or [9], which we can only imagine what they were using mathse for 👎 )(sorry for the rant) 

I don't have comments on section 8.  But I remember reading reference [7] before and thinking (especially after looking at the affiliations of the myriad of authors) this just seems like a big advertisement designed to showcase the capabilities of Deepmind (subsidiary of Google).  For sure they have done impressive things (AlphaGo, etc), but at the moment it just seems a bunch of hype as far as usage in mathematical research.  Just my impression.

**Reply by StevenClontz** [reply]
> Def 50: was the term "anti-locally compact" made up for this article? Annoyingly, it's unrelated to the terminological scheme from Bankston - "the total negation of a topological property" (https://doi.org/10.1215/ijm/1256048236).

It was. Probably "nowhere locally compact" is better here.

**Reply by StevenClontz** [reply]
References: I'm usually very lazy with references until I'm working with a technical editor for the finalized draft, since they can have Opinions. Or if the referee has opinions, of course. So these are mostly the output from Google Scholar or MathSciNet unedited.

The StackExchange network citations include those IDs automatically; I will look at them closer.

**Reply by StevenClontz** [reply]
I agree that most AI/LLM stuff is overhyped. But it's in the zeitgeist so I feel a need to address it. It's not often I get to cite *Nature* in my work anyway so...

**Reply by StevenClontz** [reply]
Thanks for these notes. I think I ended up making changes for each.

**Reply by StevenClontz** [reply]
> > Def 50: was the term "anti-locally compact" made up for this article? Annoyingly, it's unrelated to the terminological scheme from Bankston - "the total negation of a topological property" (https://doi.org/10.1215/ijm/1256048236).
> 
> It was. Probably "nowhere locally compact" is better here.

Hmm it seems that is the right choice, see e.g.
http://topology.nipissingu.ca/tp/reprints/v59/tp59005p1.pdf (though as usual, some thought would need to be put into compact neighborhoods vs. closure of open is compact without higher separation axioms, before contributing it to pi-Base)

**Comment by ccaruvana** [comment]
I have nothing of substance to add other than I'm excited about it. Thanks for this contribution, Steven!

At the top of page 11, you have "Fortunately, recent advances in GitHub services suggest a near future where these technical can be mostly abstracted away, ..." and I think you intend for a word to follow "technical" or maybe you meant to use "technicalities."

**Reply by StevenClontz** [reply]
Thanks - this is now fixed.

**Comment by prabau** [comment]
Related to Alan Dow's comment in the talk: in Defs 41 and 42, it should be "regular open" instead of "open regular".

**Reply by StevenClontz** [reply]
Done, thanks

**Comment by dimpase** [comment]
The title strikes me as somewhat unbalanced, and not doing justice to the main tool used, i.e.  $\pi$-Base (To me, titles such as "Foo and Bar" are meant for contributions to Foo and Bar being somewhat comparable, a least). 

How about something like "Non-Hausdorff $T_1$ Properties: from the shoulders of $\pi$-Base and other 21st century sociotech"

**Reply by dimpase** [reply]
(PS. As you see, you can use $\TeX$ in GitHub comments, the usual markup with dollar signs  :-))

**Reply by StevenClontz** [reply]
To be honest, I do somewhat see the two topics as being of balanced interest in this paper. I tried to write it so folks just interested in the mathematical results could get something out of it, while also integrating the use of $\pi$-base throughout to motivate the last section on how technology changes and can change how we do our research. In particular, I think that the current title is more likely to play well with reviewers, but I could be mistaken.
